### PR TITLE
feat(veeam-backup): listing page

### PR DESCRIPTION
**Pull request by chipp972**
ref: MANAGER-14884

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/hpc-veeam-backup` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-14884 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by rjamet-ovh — APPROVED** [review]

File: packages/manager/apps/veeam-backup/src/pages/listing/Listing.page.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    data,
+    fetchNextPage,
+    hasNextPage,
+    isError,
+    isLoading,
+    error,
+    status,
+  } = useVeeamBackupList({ pageSize: 10 });
+  const { data: regions } = useRegions();
+  const { data: organizations } = useOrganizations();
```
Comment (anooparveti) [inline packages/manager/apps/veeam-backup/src/pages/listing/Listing.page.tsx:47]: **nitpick:** You can directly use `useIcebergV2` hook from `manager-components` to fetch the data.

**Review by anooparveti — CHANGES_REQUESTED** [review]
Folder structure is not well maintained in the `data` folder. It must only have `api` and `hook` folder.

File: packages/manager/apps/veeam-backup/public/translations/listing/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+  "description": "Gérer les sauvegardes de vos organisations VMware Cloud Director",
+  "orderButton": "Commander",
+  "nameCell": "Nom",
+  "statusCell": "Statut",
+  "ovhrefCell": "Référence",
+  "vcdorgCell": "Organisation VCD",
+  "regionCell": "Région",
+  "createdatCell": "Date de création",
+  "deleteAction": "Supprimer",
+  "loadMoreButton": "Charger les sauvegardes suivantes"
```
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/veeam-backup/public/translations/listing/Messages_fr_FR.json]: Done fixed

File: packages/manager/apps/veeam-backup/src/App.tsx [hunk 3]
```diff
@@ -19,14 +23,34 @@ const queryClient = new QueryClient({
...
+  const [hiddenMessages, setHiddenMessages] = React.useState([]);
+
+  const messageContext = React.useMemo(
+    () => ({
+      successMessages,
+      hiddenMessages,
+      addSuccessMessage: (msg: string, options: MessageOptions) => {
+        setSuccessMessages((msgList) =>
+          msgList.concat({ id: Date.now(), message: msg, options }),
+        );
```
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/App.tsx]: Done fixed

File: packages/manager/apps/veeam-backup/src/pages/listing/Listing.page.tsx (hunk 1 above)
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/pages/listing/Listing.page.tsx]: done fixed

File: packages/manager/apps/veeam-backup/src/data/veeam-backup/veeam-backup.hook.ts
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/data/veeam-backup/veeam-backup.hook.ts:1]: done fixed

File: packages/manager/apps/veeam-backup/src/data/vcd-region.hook.ts
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/data/vcd-region.hook.ts:1]: done fixed

File: packages/manager/apps/veeam-backup/src/data/vcd-org.hook.ts
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/data/vcd-org.hook.ts:1]: done fixed

File: packages/manager/apps/veeam-backup/src/App.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/App.tsx]: done fixed

File: packages/manager/apps/veeam-backup/public/translations/listing/Messages_fr_FR.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (chipp972) [reply packages/manager/apps/veeam-backup/public/translations/listing/Messages_fr_FR.json]: done fixed

**Review by pauldkn — APPROVED** [review]

**Pull request merged**